### ISSUE TITLE
Dial tcp: lookup apprunner.eu-central-1.amazonaws.com on 10.253.0.10:53: no such host

### ISSUE THREAD

**Issue by glebobos** [devopsbox-io/aws-ecr-cleaner#2]
Please take a look https://docs.aws.amazon.com/general/latest/gr/apprunner.html , there is no eu-central-1 region and may be I can somehow disable apprunner?

**Comment by mraszplewicz** [comment]
@glebobos 

Thanks for reporting this issue!

I've added checking if App Runner is supported in a region by using SSM GetParametersByPath. With AWS CLI it would be:
```shell
aws ssm get-parameters-by-path \
  --path /aws/service/global-infrastructure/services/apprunner/regions
```

It is in the [bugfix/region-without-apprunner](https://github.com/devopsbox-io/aws-ecr-cleaner/compare/bugfix/region-without-apprunner) branch. I still have to perform some tests before releasing it. It would be great if you could build it yourself and check in your environment.

**Comment by mraszplewicz** [comment]
@glebobos 

I've just released aws-ecr-cleaner v0.1.1 - this bug should be fixed but you have to update your Lambda IAM policy and add the following statement:
```json
{
  "Action": [
    "ssm:GetParametersByPath"
  ],
  "Effect": "Allow",
  "Resource": [
    "arn:aws:ssm:*::parameter/aws/service/global-infrastructure/services/apprunner/regions"
  ]
}
```
